src/components/BibPage/BibDetail.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  Link as DSLink,
+  Heading,
+  List,
+} from "@nypl/design-system-react-components"
+import styles from "../../../styles/components/BibDetails.module.scss"
+import RCLink from "../RCLink/RCLink"
+import type {
+  BibDetail,
+  Url,
+  LinkedBibDetail,
+  SubjectHeadingDetail,
+} from "../../types/bibTypes"
+import { displayRtl, isItTheLastElement } from "../../utils/bibDetailUtils"
+
+interface BibDetailsProps {
+  details: BibDetail[] | LinkedBibDetail[]
+  heading?: string
+}
+
+const BibDetails = ({ details, heading }: BibDetailsProps) => {
+  return (
+    details?.length > 0 && (
+      <>
+        {heading && <Heading level="three">{heading}</Heading>}
+        <List
+          type="dl"
+          className={styles.bibDetails}
+          sx={{ borderBottom: "none" }}
+        >
+          {details.map(
+            (detail: BibDetail | LinkedBibDetail | SubjectHeadingDetail) => {
+              if (!detail) return
+              if (detail.label === "Subjects") {
+                return buildCompoundSubjectHeadingElement(
+                  detail as SubjectHeadingDetail
+                )
+              } else if ("link" in detail) {
+                return buildLinkedElement(detail as LinkedBibDetail)
+              } else {
+                return buildDetailElement(detail as BibDetail)
+              }
+            }
+          )}
+        </List>
+      </>
+    )
+  )
+}
+
+const buildDetailElement = (field: BibDetail) => {
+  return (
+    <>
+      <dt>{field.label}</dt>
+      <dd>
+        <List data-testId={`${field.label}`} type="ol">
```

Comment:
Regarding the question in the PR's description: why the use of `ol`? The current page uses `ul` and I'm not sure if the values for each field should be numbered.

Reply:
I'm using ol because the order matters for the primary/parallel matches. but maybe i'm misinterpreting the use case for an ol. happy to use whatever!